Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_050.IdentifiableNpmAuthorToken.ps1

```diff
@@ -0,0 +1,3 @@
+﻿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
+
+"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"
```

Comment:
>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 [](http://example.com/codeflow?start=1&length=40)

here, you created a test that pass the checksum.
Can you also add a new one that do not pass?
Also, add a comment. Something like:

# This is a pattern that matches the regex but the checksum is invalid
npm_abcd....